internal/cli/task_cancel.go

```diff
@@ -0,0 +1,103 @@
+package cli
+
+import (
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	pb "github.com/hashicorp/waypoint/pkg/server/gen"
+)
+
+type TaskCancelCommand struct {
+	*baseCommand
+
+	flagRunJobId string
+}
+
+func (c *TaskCancelCommand) Run(args []string) int {
+	// Initialize. If we fail, we just exit since Init handles the UI.
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(c.Flags()),
+	); err != nil {
+		return 1
+	}
+	ctx := c.Ctx
```

Comment:
nit: we don't seem to get any value out of this statement as we don't appear to do anything with `ctx` beyond pass it into another method. Is there a good reason to do this here?

Reply:
Yeah, we can keep it to just use `c.Ctx`. There's no explicit reason here other than that's a similar pattern for other CLI commands

Reply:
I saw some that did, and some that didn't, which is why I asked. Feel free to leave as-is